nyaa/templates/layout.html

```diff
@@ -90,6 +90,7 @@
 						{% endif %}
 						{% if g.user.is_moderator %}
 							<li><a href="{{ url_for('view_reports') }}">Reports</a> </li>
+                            <li><a href="{{ url_for('view_adminlog') }}">Log</a> </li>
```

Comment:
@mreweilk 
1. The bug with the nav bar (wrapping to a new line) is already there ever since the Reports were added to it.
2. About the screenshot, sorry, I was being lazy... haha.
Attached is a new, accurate screenshot.

    Notice the width - 992px - this is the most narrow view the site has before it collapses to the menu on the side.
![image](https://user-images.githubusercontent.com/10238474/27212357-21d3cf88-5268-11e7-880c-b94dad3b79e2.png)

3. I believe I found a fix for the issue.
I'll open a pull request on your fork.